tests/components/BidHistory.test.tsx

```diff
@@ -153,7 +154,7 @@ describe("AuctionInfo", () => {
       );
 
       const wonArea = await screen.findByText("won the auction");
-      expect(wonArea.parentElement?.innerHTML).toContain("June 11, 12:25 PM");
+      expect(wonArea.parentElement?.parentElement?.parentElement?.innerHTML).toContain("June 11, 12:25 PM");
```

Comment:
same

Reply:
Changed to `await screen.findAllByText("June 11, 12:25 PM");` as there are two instances of the datestamp.